web-search-engines/langchain4j-web-search-engine-searchapi/src/test/java/dev/langchain4j/web/search/searchapi/SearchApiWebSearchContentRetrieverIT.java

```diff
@@ -0,0 +1,61 @@
+package dev.langchain4j.web.search.searchapi;
+
+import dev.langchain4j.model.chat.ChatLanguageModel;
+import dev.langchain4j.model.openai.OpenAiChatModel;
+import dev.langchain4j.rag.content.retriever.WebSearchContentRetriever;
+import dev.langchain4j.rag.content.retriever.WebSearchContentRetrieverIT;
+import dev.langchain4j.service.AiServices;
+import dev.langchain4j.web.search.WebSearchEngine;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.condition.EnabledIfEnvironmentVariable;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static dev.langchain4j.web.search.searchapi.SearchApiWebSearchEngine.DEFAULT_ENV_VAR;
+
+
+@EnabledIfEnvironmentVariable(named = "SEARCHAPI_API_KEY", matches = ".+")
+class SearchApiWebSearchContentRetrieverIT extends WebSearchContentRetrieverIT {
```

Comment:
This test fails for me with `java.util.concurrent.CompletionException: java.lang.IllegalArgumentException: The metadata value for the key 'thumbnail' cannot be null`, but the response from Google is coming. Looks like API has been changed in the meanwhile? 🤔

Reply:
test still fails with the same exception

Reply:
still fails, same exception

Reply:
Would you be kind enough to include the full stack trace?

Thanks!

Reply:
Should be fixed now. Please kindly confirm.

Reply:
Seems to work